.circleci/main.yml

```diff
@@ -1480,6 +1480,43 @@ jobs:
             PYTHON_SDK_TESTING_IMAGE=pachyderm/python-sdk-ci-testing:$CIRCLE_SHA1 \
             poetry run pytest -vvv tests
 
+  python-sdk-test-publish:
+    executor: python
+    working_directory: ~/project/python-sdk
+    parameters:
+      version:
+        type: string
+    steps:
+      - checkout:
+          path: ~/project
+      - attach_workspace:
+          at: .
+      - install-poetry
+      - run:
+          name: Publish
+          command: |
+            poetry config repositories.test-pypi https://test.pypi.org/legacy/
+            poetry version << parameters.version >>
```

Comment:
will this not cause failures on ` python-sdk-publish` trying to publish duplicate version?

Reply:
No, this publishes to `test.pypi.org` instead of `pypi.org`. Testing a release with the same version on there is expected functionality